src/api/procedures/inviteAccount.ts

```diff
@@ -10,6 +18,11 @@ import {
 
 export interface InviteAccountParams {
   targetAccount: string | Account;
+  permissions?: {
+    tokens?: (string | SecurityToken)[];
+    transactions?: TxTag[];
+    portfolios?: (DefaultPortfolio | NumberedPortfolio)[];
+  };
```

Comment:
You should be able to pass null values as well (to reflect giving all permissions)

```
{
  tokens: null,
  transactions: null,
  portfolios: null,
}
```

This should be reflected on the comments for `inviteAccount` as well